CosNet.WebUI/CosNet.WebUI.csproj

```diff
@@ -30,6 +30,9 @@
   <ItemGroup>
     <ComponentScssFiles Include="**/*/*.scss" Exclude="node_modules/**;wwwroot/**;Styles/**" />
   </ItemGroup>
+  <ItemGroup>
+    <Watch Remove="Styles\_cosplay-item-material-list.scss" />
```

Comment:
deze watch remove mag terug verwijderd omwille van dotnet watch run

Reply:
yeah forgot about it